packages/knip/fixtures/plugins/cypress-builtin-reporter/cypress/support/commands.ts

```diff
@@ -0,0 +1,7 @@
+import { faker } from '@faker-js/faker';
+
+function login() {
+  return faker;
+}
+
+Cypress.Commands.add('login', login);
```

Comment:
Same, no need for unnecessary files in fixture folders (less bloat is better long-term).